### ISSUE TITLE
How do you define Regions ?

### ISSUE THREAD

**Issue by Raymont99** [thecolonel63/server-side-replay-recorder#31]
Hey, 
First of all i wanna say a huge thanks for making this mod, been looking for a way to record longer replays server sided for some time now, since we do bigger projects on our server.

I am sorry if this a noob question, but i couldnt find any information on the wiki or online, but how do you define a region ?

Thank you in advance

**Comment by Cyclopropinon** [comment]
yeah a sort of small wiki would be very nice, especially for "beginners"

**Comment by mattymatty97** [comment]
the command to start a region recording is:
`/replay region [region name] start [cornerx] [cornerz] [corner2x] [corner2z]`

can also be interpeted as

`/replay region [region name] start [col1] [col2]`

the corners can be tab completed by looking at the corner blocks
( i copied the code from the forceload command )

well all commands have tab competition for easier compilation

REMEMBER: the coords in the command are block clumn coords but the mod internally will convert them to chunks
and the status command will show chunk coords instead

PS:
if you have to start a recording from the  console wrap the command into:
`/execute in [dimension] run [replay command]` as the command won't know the dimension to record from and will crash, while if executed by a player it will use the player dimension


have a nice recording :smile:

**Comment by thecolonel63** [comment]
Will keep this open until a proper wiki with an in-depth overview of everything about the mod exists eventually.

**Comment by Raymont99** [comment]
Thank you so much for your quick response ! 
Works like a charm :)
